File: ruby/test-fluent.rb
require 'rubygems'
require 'bundler/setup'

require 'logstash-logger'

tcp_logger = LogStashLogger.new(type: :tcp, host: '127.0.0.1', port: 5170, sync: true)

for i in 0..10000 do
    tcp_logger.info 'test'
end

File: nodejs/test-fluent.mjs
import Transport from 'winston-tcp'
import winston from 'winston'

let logger = new(winston.Logger)({
  transports: [
    new(Transport)({
      host: '127.0.0.1',
      port: 5170,
      json: true,
      timestamp: true
    })
  ]
})

for (var i = 0; i < 200000; i++) {
  logger.log('info', 'foo')
}